docs/wip/how-to-rollback.md

```diff
@@ -1,12 +1,115 @@
-## Step by step on how to rollback?
+## ⚠️ Warning: this will only work when we start using VTEX infrastructure to build the images. Is also possible to build with Netlify[] — for rollback in Netlify, you can use it's tools.
 
 
-- How much time it takes?
-- security measures
+# The rollback
+The rollback interaction aims to help you to react faster when having trouble, changing the deploy that is on production thought a safe and tested path. This also allows you to reuse older versions of a storefront, for a recurrent store's campaing for example. 
 
-- How the rollback works with dynamic and static data?
-- Caching? How much time it takes to invalidate caching
+# 📚 Step by step on how to rollback
+This document is the first version of a guide for the rollback process using VTEX Store Framework Jamstack. It also explains the rollback expected behavior.
+We'll start showing how to make a safe deploy — one that is ready to be used for a rollback.
 
+## 👮‍♀️ How to make a safe deploy (Security Measures)
+- Create a new branch from the current master (this is a backup branch):
```

Comment:
```suggestion
# Step by step

## Step 1: Making a safe deploy

In case you need to do a rollback in the future, we always recommend you create a backup branch of the `master` as a security measure. For that, take the following steps:

1.  Create a backup branch from the current master:

```